### DISCUSSION TITLE
CouchDB releases shows old tagged version 2.3.0

### DISCUSSION THREAD

**Discussion by big-r81** [apache/couchdb#3002] [General] [answered]
Hi all,

the latest tagged version of CouchDB is 2.3.0 at the github repo (https://github.com/apache/couchdb). This should be updated to the latest version.

**Comment by wohali** [comment]
Hey @big-r81 ,

We don't use GitHub for releases, sorry.

The tags are all present in the repo:

* https://github.com/apache/couchdb/tree/3.1.0
* https://github.com/apache/couchdb/tree/3.0.1

etc.

**Comment by big-r81** [comment, marked as answer]
It was only a hint that an old version as the latest is tagged...

**Reply by wohali** [reply]
Thanks for the heads up, I removed all of those tagged releases from the sidebar.
